cpplib/Tests/MainTest.cpp

```
@@ -93,6 +97,259 @@ struct FMIC_TS {
+
+			path_i++;
+		}
+	}
+}
+
+
+TEST(VoronoiTest, Crash1) {
+	using Diagram = cpplib::voronoi::VoronoiDiagram;
+	using BaderOperator = cpplib::BaderOperator;
+	std::array<cpplib::basic_types::FloatingPointType, 6> cell_in{30.75, 20.975, 15.835, 90.0, 90.0, 90.0};
+	std::vector<const char*> symm{{"x, y, z", "-x+1/2, -y, z+1/2", "-x, y+1/2, -z", "x+1/2, -y+1/2, -z+1/2", "-x, -y, -z", "x-1/2, y, -z-1/2", "x, -y-1/2, z", "-x-1/2, y-1/2, z-1/2"}};
```

Clang (14.0.6)

[warning] variable 'symm' is not initialized

(cppcoreguidelines-init-variables)